.github/workflows/delete-release.yml

```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   purge-image:
     name: Delete image from ghcr.io
-    runs-on: ubuntu-22.04
+    runs-on: self-hosted
```

Comment:
I would say that this action should remain on the default GHA Runners, as it does not interact with the source code either.